### DISCUSSION TITLE
Improve color picker viewport display

### DISCUSSION THREAD

**Discussion by pedrorrodriguez** [aurelienpierreeng/ansel#781] [Ideas]
Back in the day I took a look at Raw therapee, and its way of handling the color picker tool is partly better than in Ansel. When we pick several values, and add them to the list in the color picker module, a crosshair is placed on the viewport, while the values are displayed in the module itself. 

Raw therapee displays the swatch on the picture, as well as its values. This makes it easier to identify where each swatch was sampled from.

![imaxe](https://github.com/aurelienpierreeng/ansel/assets/64545422/a2093c10-e69b-49d4-ba45-9593fa8abd79)

They don't need to be displayed all the time, but it seems helpful if you use the color picker. We also don't need to display one swatch in RGB, another in HSV, and another in Lab. Just whatever is selected in the color picker module.

TLDR, keep what we have, but make it possible to display the values directly on the picture as well.
